guides/extensions/push-notifications/index.md

```diff
@@ -0,0 +1,3 @@
+# Push Notifications plugin for Aries Framework JavaScript
+
+The Push Notifications package provides a simple way to implement push notications in a client application. It collects events triggered by the agent via AFJ into simple APIs that you can use to trigger push notiifcations in your client application.
```

Comment:
This description is not fully correct. 

It provides a way to register your apns and firebase push notification token at another agent (most probably a mediator)

Reply:
Ah, got it. Fixing now.